tests/Go/Functional/BaseFunctionalTest.php

```diff
@@ -40,7 +43,8 @@ protected static function exec($command, $args = '')
 
         $process->run();
 
-        self::assertTrue($process->isSuccessful(), sprintf('Unable to execute "%s" command.', $command));
+        $assert = ($expectSuccess) ? 'assertTrue' : 'assertFalse';
+        self::{$assert}($process->isSuccessful(), sprintf('Unable to execute "%s" command, got output: "%s".', $command, $process->getOutput()));
```

Comment:
Better not to use dynamic static method, just write if..else construction, it will be more readable